packages/docusaurus/lib/load/index.js

```diff
@@ -23,7 +23,6 @@ module.exports = async function load(siteDir, cliOptions = {}) {
     siteDir,
     constants.GENERATED_FILES_DIR_NAME,
   );
-  fs.ensureDirSync(generatedFilesDir);
```

Comment:
No need to call this since below line

https://github.com/facebook/Docusaurus/blob/f1739abab313d5ad9a1756cba4cc683c720d490d/packages/docusaurus/lib/load/index.js#L27-L33

will definitely call it

https://github.com/facebook/Docusaurus/blob/5db02f41a95903b2e898bbf625007929195ff41f/packages/docusaurus-utils/src/index.js#L16-L23